fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/type/NewQuantityParmBehaviorUtil.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.util.type;
+
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.CODE;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.CODE_SYSTEM_ID;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.QUANTITY_VALUE;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import com.ibm.fhir.database.utils.query.WhereFragment;
+import com.ibm.fhir.persistence.exception.FHIRPersistenceException;
+import com.ibm.fhir.persistence.jdbc.dao.api.JDBCIdentityCache;
+import com.ibm.fhir.persistence.jdbc.util.CodeSystemsCache;
+import com.ibm.fhir.search.SearchConstants.Prefix;
+import com.ibm.fhir.search.parameters.QueryParameter;
+import com.ibm.fhir.search.parameters.QueryParameterValue;
+
+/**
+ * Quantity parameter handling using the Select model.
+ *
+ * <a href="https://hl7.org/fhir/search.html#quantity>FHIR Search - Quantity</a>
```

Comment:
Missing a double-quote after the url causes review to lose context highlighting.

Reply:
ok